spec/amp-var-substitutions.md

```diff
@@ -32,422 +29,868 @@ will still update to the most current values of the variables.  Note that some
+#### Canonical URL
 
-Use the special string `SOURCE_URL` to add the source URL of the current document to the URL.
+Provides the canonical document's URL.
 
-The source URL is extracted from the proxy URL if the document is being served from a proxy. Otherwise the original document URL is returned. For instance, if the URL is served via the proxy `https://cdn.ampproject.org` from the URL `https://cdn.ampproject.org/c/s/example.com/page.html`, then `SOURCE_URL` would return `https://example.com/page.html`.
+* **platform variable**: `CANONICAL_URL`
+  *  Example: <br>
+  ```html
+  <amp-pixel src="https://foo.com/pixel?href=CANONICAL_URL"></amp-pixel>
+  ```
+  Makes a request to something like `https://foo.com/pixel?href=https%3A%2F%2Fpinterest.com%2F`.
+* **amp-analytics variable**: `${canonicalUrl}`
+  * Example value: `http%3A%2F%2Fexample.com%3A8000%2Fanalytics.html`
 
-For instance:
-```html
-<amp-pixel src="https://foo.com/pixel?href=SOURCE_URL"></amp-pixel>
-```
-may make a request to something like `https://foo.com/pixel?href=https%3A%2F%2Fpinterest.com%2F`.
+#### Client ID
 
-### SOURCE_HOST
+Provides a per document-source-origin (the origin of the website where you publish your AMP doc) and user identifier. The client ID will be the same for the same user if they visit again within one year. The client ID should behave roughly similar to a cookie storing a session ID for one year. If the AMP document is not served through the Google AMP Cache, the client ID is replaced with a cookie of the name of the `cid scope` argument (see below). If it is not present, a cookie will be set with the same name. These cookies will always have the prefix "amp-" followed by a random base64 encoded string.
 
-Use the special string `SOURCE_HOST` to add the source URL's host of the current document to the URL. See the description of `SOURCE_URL` for more details.
+* **platform variable**: `CLIENT_ID`
+  *  Example: <br>
+  ```html
+  <amp-pixel src="https://foo.com/pixel?cid=CLIENT_ID(cid-scope-cookie-fallback-name)"></amp-pixel>
+
+<amp-user-notification
+    layout=nodisplay
+    id="user-consent"
+    data-show-if-href="https://foo.com/api/show"
+    data-dismiss-href="https://foo.com/api/dismissed">
+    This site uses cookies to personalize content.
+    <a href="">Learn more.</a>
+   <button on="tap:user-consent.dismiss">I accept</button>
+</amp-user-notification>
 
-For instance:
-```html
-<amp-pixel src="https://foo.com/pixel?host=SOURCE_HOST"></amp-pixel>
-```
-may make a request to something like `https://foo.com/pixel?host=pinterest.com:9000`.
+<!-- cid is not provided until `user-consent` is dismissed -->
+<amp-pixel src="https://foo.com/pixel?cid=CLIENT_ID(cid-scope-cookie-fallback-name,user-consent-id)"></amp-pixel>
+  ```
+* **amp-analytics variable**: `${clientId}`
+  * Example usage: `${clientId(foo)}`
+  * Example value: `U6XEpUs3yaeQyR2DKATQH1pTZ6kg140fvuLbtl5nynbUWtIodJxP5TEIYBic4qcV`
 
-### SOURCE_HOSTNAME
+**Arguments**
```

Comment:
In the rendered doc, the 'arguments' and client id look exactly the same and it looks like arguments is just another type of variable. If there any way to change the formatting?